src/jdk.jdwp.agent/share/native/libjdwp/invoker.c

```diff
@@ -790,23 +792,20 @@ invoker_completeInvokeRequest(jthread thread)
         (void)outStream_writeValue(env, &out, tag, returnValue);
         (void)outStream_writeObjectTag(env, &out, exc);
         (void)outStream_writeObjectRef(env, &out, exc);
+        /*
+         * Delete potentially saved global references for return value
+         * and exception. This must be done before sending the reply or
+         * these objects will briefly be viewable by the debugger as live
+         * when they shouldn't be.
+         */
+        if (mustReleaseReturnValue && returnValue.l != NULL) {
+          tossGlobalRef(env, &returnValue.l);
```

Comment:
Please make indentation 4 spaces to be consistent with other code in the file.
The same for line 805

Reply:
Fixed.